src/rainbow-fetch/index.ts

```diff
@@ -91,6 +91,7 @@ function generateError({
 }) {
   const message =
     responseBody?.error ||
+    responseBody?.message ||
```

Comment:
we often use "message" in our RainbowErrors and without this, it was often falling back to the generic "There was an error with the request"